releases/release-8.1.1.md

```diff
@@ -0,0 +1,198 @@
+---
+title: TiDB 8.1.1 Release Notes
+summary: Learn about the compatibility changes, improvements, and bug fixes in TiDB 8.1.1.
+---
+
+# TiDB 8.1.1 Release Notes
+
+Release date: August xx, 2024
+
+TiDB version: 8.1.1
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v8.1/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v8.1/production-deployment-using-tiup)
+
+## Compatibility changes
+
+- When using TiDB Lightning to import a CSV file, if you set `strict-format = true` to split a large CSV file into multiple small CSV files to improve concurrency and import performance, you need to explicitly specify `terminator`. The values can be `\r`, `\n` or `\r\n`. Failure to specify a line terminator might result in an exception when parsing the CSV file data. [#37338](https://github.com/pingcap/tidb/issues/37338) @[lance6716](https://github.com/lance6716)
+- When using [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) to import a CSV file, if you specify the `SPLIT_FILE` parameter to split a large CSV file into multiple small CSV files to improve concurrency and import performance, you need to explicitly specify the line terminator `LINES_TERMINATED_BY`. The values can be `\r`, `\n` or `\r\n`. Failure to specify a line terminator might result in an exception when parsing the CSV file data. [#37338](https://github.com/pingcap/tidb/issues/37338) @[lance6716](https://github.com/lance6716)
```

Comment:
```
- Changed the default value of the variable tidb_enable_parallel_hashagg_spill to false. [#55290](https://github.com/pingcap/tidb/issues/55290) @[xzhangxian1008](https://github.com/xzhangxian1008)
```

Reply:
added in [`d3b6b11` (#18537)](https://github.com/pingcap/docs/pull/18537/commits/d3b6b114ddef6bf47113f1cfad9ab3c44e943a54)